tutorials/create-cloud-build-image-factory-using-packer/index.md

```diff
@@ -114,82 +370,608 @@ To see a list of image families:
 
     gcloud compute images list | awk '{print $3}'  | awk '!a[$0]++'
 
-## Add the packer Cloud Build image to your project
+</details>
+
+## Building a Windows Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
+You can create a trigger on the [build triggers page](https://console.cloud.google.com/cloud-build/triggers)
+of the Cloud Console by following these steps:
+
+1.  Click **Create Trigger**.
+1.  In the **Name** field, enter `Windows image factory`.
+1.  Under **Event**, select **Push new tag**.
+1.  Under **Source**, select `windows-image-factory` as your
+    **Repository** and the tag to match or .* (any tag) as your **Tag**.
+1.  Under **Build Configuration**, select **Cloud Build configuration file (yaml or json)**.
+1.  In the **Cloud Build configuration file location**, enter `cloudbuild.yaml`.
+1.  Click **Create** to save your build trigger.
+
+</details>
+
+# Task 7: Add the packer Cloud Build image to your project
+
+Get the builder from the community repository and submit it to your project. This will allow Cloud Build to use a docker container that contains the packer binaries.
+
+## Building a Linux Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```sh
+project_dir=$(pwd)
+cd /tmp
+git clone https://github.com/GoogleCloudPlatform/cloud-builders-community.git
+cd cloud-builders-community/packer
+gcloud builds submit --config cloudbuild.yaml
+rm -rf /tmp/cloud-builders-community
+cd $project_dir
+```
 
-Get the builder from the community repository and submit it to your project:
+</details>
 
-    project_dir=$(pwd)
-    cd /tmp
-    git clone https://github.com/GoogleCloudPlatform/cloud-builders-community.git
-    cd cloud-builders-community/packer
-    gcloud builds submit --config cloudbuild.yaml
-    rm -rf /tmp/cloud-builders-community
-    cd $project_dir
+## Building a Windows Image
 
-## Add your repository as a remote repository and push
+<details>
+<summary>Click to reveal commands</summary>
 
-1.  (Only if not running in Cloud Shell) Set up your Google credentials for git:
+```powershell
+$env:PROJECT_DIR=$(Get-Location)
+New-Item -Path "C:\" -Name "temp" -ItemType Directory
+Set-Location -Path "C:\temp"
 
-        gcloud init && git config --global credential.https://source.developers.google.com.helper gcloud.sh
+git clone https://github.com/GoogleCloudPlatform/cloud-builders-community.git
+Set-Location -Path "./cloud-builders-community/packer"
+gcloud builds submit --config cloudbuild.yaml
 
-2.  Add the `google` repository as a remote:
+Remove-Item -Path "C:\temp\cloud-builders-community" -Recurse -Force
+Set-Location -Path $env:PROJECT_DIR
+```
 
-        git remote add google \
-          https://source.developers.google.com/p/$PROJECT/r/helloworld-image-factory
+</details>
 
-## Task 9: Push the repository and tags to google
+# Task 8: Add your repository as a remote repository and push
```

Comment:
I think this is beyond the scope of this tutorial -- I'd prefer to remove it to keep it shorter

Reply:
I combined Task 8 and 9 together (push local files to remote repo) to show an end to end CI/CD pipeline (which was the original purpose of the image factory)